content/es/docs/getting-started/demos-talks-posts/articles.md

```diff
@@ -0,0 +1,52 @@
+---
```

Comment:
I don't think it makes sense to translate this page, unless someone wants to commit to keeping it translated continuously.

Reply:
Yes, agreed.  @mmorejon maybe these can stay the way they are?  Also less work maintaining the translation if we don't bother translating these...

Reply:
Good. This case is similar to the question asked in the following conversation.